src/vendor-styles.css

```
@@ -1,3 +1,4 @@
+@import url("https://fonts.googleapis.com/icon?family=Material+Icons");
```

Stylelint (17.12.0)

[error] Expected "url("https://fonts.googleapis.com/icon?family=Material+Icons")" to be ""https://fonts.googleapis.com/icon?family=Material+Icons"" (import-notation)

(import-notation)